wpilibc/src/main/native/include/frc/kinematics/DifferentialDriveOdometry.h

```diff
@@ -79,6 +90,20 @@ class DifferentialDriveOdometry {
                                const Rotation2d& gyroAngle,
                                const DifferentialDriveWheelSpeeds& wheelSpeeds);
 
+  /**
+   * Updates the robot position on the field using distance measurements from
+   * encoders. This method is more numerically accurate than using velocities to
+   * integrate the pose and is also advantageous for teams that are using lower
+   * CPR encoders.
+   *
+   * @param gyroAngle The angle reported by the gyroscope.
+   * @param leftDistanceMeters The distance traveled by the left encoder.
```

Comment:
The parameter names in the comment are wrong; they don't match the real arguments.